examples/Jenkins/Sample/backstop_data/engine_scripts/puppet/clickAndHoverHelper.js

```diff
@@ -8,6 +9,11 @@ module.exports = async (page, scenario) => {
     await page.hover(hoverSelector);
   }
 
+  if (focusSelector) {
```

Comment:
Just curious here -- any reasoning with ordering after hover and before click?

Reply:
No real reason other than a mental model of how I view events (hover and focus more likely to offer small UI updates of intent, and clicks actually doing something). In my experience from using it on our projects, we used hover _or_ focus on our tests, (one simulating mouse, the other a keyboard) so it was more just ensuring it came before the click.